tests/e2e/const.go

```diff
@@ -103,6 +105,111 @@ var (
 type Version = string
 
 const (
+
 	// LocalVersion means using the files in the local repository snapshot.
 	LocalVersion Version = "local"
+
+	kubectlArgGoTemplateName                              = `-o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}'`
+	kubectlArgGoTemplateKindNameNamespace                 = `-o=go-template='{{range .items}}{{.kind}}{{"/"}}{{.metadata.name}}{{if .metadata.namespace}}{{"."}}{{.metadata.namespace}}{{end}}{{"\n"}}{{end}}'`
+	kubectlArgGoTemplateInternalListenersName             = `-o=go-template='{{range $key,$value := .status.listenerStatuses.internalListeners}}{{$key}}{{"\n"}}{{end}}`
+	kubectlArgGoTemplateInternalListenerAddressesTemplate = `-o=go-template='{{range .status.listenerStatuses.internalListeners.%s}}{{.address}}{{"\n"}}{{end}}`
+	// kubectlArgGoTemplateExternalListenersName             = `-o=go-template='{{range $key,$value := .status.listenerStatuses.externallListeners}}{{$key}}{{"\n"}}{{end}}`
+	// kubectlArgGoTemplateExternalListenerAddressesTemplate = `-o=go-template='{{range .status.listenerStatuses.externalListeners.%s}}{{.address}}{{"\n"}}{{end}}`
+
+	crdKind                    = "customresourcedefinitions.apiextensions.k8s.io"
+	kafkaKind                  = "kafkaclusters.kafka.banzaicloud.io"
+	kafkaTopicKind             = "kafkatopics.kafka.banzaicloud.io"
+	kafkaClusterName           = "kafka"
+	testTopicName              = "topic-test"
+	kcatPodName                = "kcat"
+	zookeeperKind              = "zookeeperclusters.zookeeper.pravega.io"
+	zookeeperClusterName       = "zookeeper-server"
+	managedByHelmLabelTemplate = "app.kubernetes.io/managed-by=Helm,app.kubernetes.io/instance=%s"
+
+	defaultDeletionTimeout                 = 20 * time.Second
+	defaultPodReadinessWaitTime            = 10 * time.Second
+	defaultTopicCreationWaitTime           = 10 * time.Second
+	kafkaClusterResourceCleanupTimeout     = 30 * time.Second
+	zookeeperClusterResourceCleanupTimeout = 60 * time.Second
+	externalConsumerTimeout                = 5 * time.Second
+	externalProducerTimeout                = 5 * time.Second
+
+	kcatPodTemplate    = "templates/kcat.yaml.tmpl"
+	kafkaTopicTemplate = "templates/topic.yaml.tmpl"
 )
+
+func basicK8sResourceKinds() []string {
+	return []string{
+		"pods",
+		"services",
+		"deployments.apps",
```

Comment:
Yes, I know prometheus is an exception and we need to delete by hand.
Maybe it is safer to use to delete CRDs for every component who knows this crd delete behavior will be changed in the future for different component.

Reply:
I checked and if you use the namepsaced=false flag then only the global resources will be listed.
After your suggestion regarding group selector I will modify my function to be capable to use this.

Reply:
I agree with you it is better to use dynamically the crd names.

Reply:
"I would use these with a parsable output format and parse the output to create these lists once at suite initialization so we are more resilient to possible resource kind changes."
In the suite initalization we cannot be sure that the components already installed  or can we?

Reply:
You are right, we can only be sure about the built-ins but not the 3rd party types, we need to check those in the cases after the installation - which could be in the setup phase for deeper cases.

Reply:
> I checked and if you use the namepsaced=false flag then only the global resources will be listed. After your suggestion regarding group selector I will modify my function to be capable to use this. e.g: kubectl api-resources --api-group "" --output name --sort-by name | wc -l --> 17 kubectl api-resources --api-group "" --output name --sort-by name --namespaced=true | wc -l --> 13 kubectl api-resources --api-group "" --output name --sort-by name --namespaced=false | wc -l --> 4

Yeah, what I meant is that the default (`--namespaced=true`) case lists the namespace bound resources while the `--namespaced=false` variant lists the global (what I call cluster-wide) resources, but we need both because technically a 3rd party app could theoretically install global resources as well, like CRDs for example.

Reply:
fix:[3addfae](https://github.com/banzaicloud/koperator/pull/987/commits/3addfae60a1074c4bed3554e2acb1b980be742f7)